src/libraries/NetCoreAppLibrary.props

```diff
@@ -145,6 +145,30 @@
       System.Xml.XPath;
       System.Xml.XPath.XDocument;
     </NetCoreAppLibrary>
+    <WindowsDesktopCoreAppLibrary>
```

Comment:
Should we follow what we did for `IsAspNetCoreApp` instead? Which is setting a `IsWindowsDesktopLibrary` in the project's `Directory.Build.props`?

Reply:
Let's actually follow what we do with NetCoreAppLibrary and move the IsAspNetCore projects into this file as well. This will help us with composing the aspnetcore internal transport package without relying on binplacing and using P2Ps instead.

Reply:
We used to have `IsAspNetCore` as a static list if I remember correctly and then moved it back, no? Or it was that we talked about moving it to this file? I can't exactly remember.

Reply:
The 5.0 branch has everything in that file: https://github.com/dotnet/runtime/blob/release/5.0/src/libraries/NetCoreAppLibrary.props. I don't we ever moved that over in the main/master branch.

Reply:
Ah, this is what I was remembering: https://github.com/dotnet/runtime/commit/a66b4e3bf5e3c5ecb2e9fba771f69b177bd3844e#diff-dd4eb818f3a25bb1e97606532166e6b30deecb4f91580b672a5527200f233795L163